client-java-contrib/cert-manager/pom.xml

```diff
@@ -0,0 +1,23 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>cert-manager</artifactId>
```

Comment:
how about `model-contrib-cert-manager` or `client-java-contrib-cert-manager`?

Reply:
yes, totally agree, I guess `client-java-cert-manager-models` is specific and intuitive enough.